images/custom-error-pages/rootfs/www/403-refresh-shedul.html

```diff
@@ -0,0 +1,20 @@
+{
+"data":{
+"attributes":{
+"application":"shedul",
+"deprecated":false,
+"locked":true,
```

Comment:
Can we make this configurable via environment variables to be able to quickly change this?